From: Jan Engelhardt <jengelh@inai.de>
Date: 2021-01-30 12:56:08.331705673 +0100

The library is not getting installed, and so must be STATIC
to avoid a DT_NEEDED entry in programs and libraries.
---
 CMakeLists.txt |    9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

Index: libwebp2-0.1.0~g350/CMakeLists.txt
===================================================================
--- libwebp2-0.1.0~g350.orig/CMakeLists.txt
+++ libwebp2-0.1.0~g350/CMakeLists.txt
@@ -485,6 +485,7 @@ install(
 # build the imageio library
 add_library(
   imageio
+  STATIC
   ${CMAKE_CURRENT_SOURCE_DIR}/imageio/anim_image_dec.cc
   ${CMAKE_CURRENT_SOURCE_DIR}/imageio/anim_image_dec.h
   ${CMAKE_CURRENT_SOURCE_DIR}/imageio/bmpdec.cc
@@ -910,18 +911,16 @@ endif()
 install(
   TARGETS webp2
   EXPORT ${PROJECT_NAME}_targets
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib)
+)
 
 if(ANDROID)
   install(
     TARGETS cpufeatures
     EXPORT ${PROJECT_NAME}_targets
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib)
+)
 endif()
 
-set(ConfigPackageLocation lib/wp2/cmake/)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/wp2/cmake/)
 
 # Install the exports.
 install(
